apps/app/src/components/ReactMarkdownComponents/NextLink.tsx

```diff
@@ -69,7 +69,7 @@ export const NextLink = (props: Props): JSX.Element => {
     const dlhref = href.replace('/attachment/', '/download/');
     return (
       <span>
-        <a id={id} href={href} className={className} target="_blank" rel="noopener noreferrer" {...dataAttributes}>
+        <a id={id} href={dlhref} className={className} target="_blank" rel="noopener noreferrer" download {...dataAttributes}>
```

Comment:
We want the pdf file to be re-displayed in the new browser tag.
I had a feeling it would download the pdf as well if I didn't.
Is there a solution to this?

Reply:
I added method to fetch content-type by href then conditionally set link href based on contentType value.
If contentType `application/pdf` the href will not modified and download set to false